src/pages/Login/Login.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useState } from 'react/cjs/react.development';
+import './Login.scss';
+
+function Login() {
+  const [loginModal, setloginModal] = useState(false);
+  const [userEmail, setuserEmail] = useState('');
+
+  const isModalOpen = e => {
+    if (setloginModal === false) {
+      setloginModal(true);
+    }
+    if (setloginModal === true) {
+      setloginModal(false);
+    }
```

Comment:
setloginModal은 loginModal state를 변경해주는 함수이기 때문에, 조건으로 들어갈 수 없습니다.

- 1차적으로는 이렇게
```suggestion
    if (loginModal === false) {
      setloginModal(true);
    }
    if (loginModal === true) {
      setloginModal(false);
    }
```
- 2차적으로는 다음처럼
```suggestion
    if (!loginModal) {
      setloginModal(true);
    }
    if (loginModal) {
      setloginModal(false);
    }
```
- 최종적으로는 이렇게 수정해볼 수 있겠네요!
```suggestion
    setloginModal(!loginModal)
```